### ISSUE TITLE
Can't connect

### ISSUE THREAD

**Issue by fe2-Nyxar** [input-leap/input-leap#1909]
### What happened?
ON nixos KDE plasma 6 Wayland: 
I get this window on the client side no matter the option i choose the windows keep re-appearing: 

![image](https://github.com/input-leap/input-leap/assets/153492661/92cd99a2-5f0e-4715-a6a6-a7eaac3db15f)

and I've looked in the logs of the server side and this is what I got:
```nix
[2024-06-06T13:34:32] INFO: starting server
[2024-06-06T13:34:32] INFO: config file: /tmp/InputLeap.LOtZjk
[2024-06-06T13:34:32] INFO: log level: INFO
[2024-06-06T13:34:33] FATAL: An error occurred: Missing libportal InputCapture portal support
[2024-06-06T13:34:33] ERROR: process exited with error code: 1
[2024-06-06T13:34:33] INFO: detected process not running, auto restarting
 ```

### Version

v2.4.0

### If applicable, where did you install Input Leap from?

from the nixpkgs

### What OSes are you seeing the problem on? (Check all that apply)

Linux

### If applicable, are you using Wayland or X11?

Wayland

### What OS versions are you using?

NIXOS

**Comment by shymega** [comment]
The correct XDG portal isn't available. This isn't an Input Leap bug but needs to be raised with the Nixpkgs maintainers. Closing as wontfix.

**Comment by fe2-Nyxar** [comment]
pluh, waiting for 4 months for the new channel to drop just to get this :d

**Comment by shymega** [comment]
Sure. I didn't want Input Leap merged into 24.05, which has now happened. It's not really Input Leap's fault, it's to do with dependencies not being the correct version in Nixpkgs. Sorry. Nothing we can do.
